cypress.json

```diff
@@ -5,7 +5,7 @@
   "projectId": "pe4h41",
   "retries": {
     "runMode": 2,
-    "openMode": 2
+    "openMode": 0
```

Comment:
@andipants12 Remember to change this back please 👍 
Set temporarily to `0` to expedite local testing with expected failures.